optimum/exporters/onnx/__main__.py

```diff
@@ -22,17 +22,10 @@
 from ...commands.export.onnx import parse_args_onnx
 from ...utils import DEFAULT_DUMMY_SHAPES, logging
 from ...utils.save_utils import maybe_save_preprocessors
+from ..error_utils import AtolError, OutputMatchError, ShapeError
 from ..tasks import TasksManager
 from .base import OnnxConfigWithPast
-from .convert import (
-    AtolError,
-    OutputMatchError,
-    ShapeError,
```

Comment:
Unless mistaken, those three are used in `__main__.py`. Why remove them?

Reply:
I import them from the proper file since it moved.